<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="8" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="582" started-at="2016-08-11T06:43:00Z" finished-at="2016-08-11T06:43:01Z">
    <groups>
      <group name="unit">
        <method signature="NewTest.aUnitTest(java.lang.String)[pri:0, instance:testnglearning.NewTest@91161c7]" name="aUnitTest" class="testnglearning.NewTest"/>
        <method signature="NewTest.cUnitTest(java.lang.String)[pri:0, instance:testnglearning.NewTest@91161c7]" name="cUnitTest" class="testnglearning.NewTest"/>
        <method signature="SecondTest.xUnitTest(java.lang.String)[pri:0, instance:testnglearning.SecondTest@5b275dab]" name="xUnitTest" class="testnglearning.SecondTest"/>
      </group> <!-- unit -->
      <group name="functional">
        <method signature="NewTest.bFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.NewTest@3c0ecd4b]" name="bFunctionalTest" class="testnglearning.NewTest"/>
        <method signature="NewTest.dFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.NewTest@3c0ecd4b]" name="dFunctionalTest" class="testnglearning.NewTest"/>
        <method signature="SecondTest.wFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.SecondTest@598067a5]" name="wFunctionalTest" class="testnglearning.SecondTest"/>
        <method signature="SecondTest.yFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.SecondTest@598067a5]" name="yFunctionalTest" class="testnglearning.SecondTest"/>
        <method signature="SecondTest.zFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.SecondTest@598067a5]" name="zFunctionalTest" class="testnglearning.SecondTest"/>
      </group> <!-- functional -->
    </groups>
    <test name="Unit Tests" duration-ms="153" started-at="2016-08-11T06:43:00Z" finished-at="2016-08-11T06:43:01Z">
      <class name="testnglearning.SecondTest">
        <test-method status="PASS" signature="xUnitTest(java.lang.String)[pri:0, instance:testnglearning.SecondTest@5b275dab]" name="xUnitTest" duration-ms="20" started-at="2016-08-11T12:13:01Z" finished-at="2016-08-11T12:13:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Manoj Kumar inside 'Unit Tests' tag]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- xUnitTest -->
      </class> <!-- testnglearning.SecondTest -->
      <class name="testnglearning.NewTest">
        <test-method status="FAIL" signature="aUnitTest(java.lang.String)[pri:0, instance:testnglearning.NewTest@91161c7]" name="aUnitTest" duration-ms="40" started-at="2016-08-11T12:13:01Z" finished-at="2016-08-11T12:13:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Manoj Kumar inside 'Unit Tests' tag]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at testnglearning.NewTest.aUnitTest(NewTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:178)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aUnitTest -->
        <test-method status="PASS" signature="cUnitTest(java.lang.String)[pri:0, instance:testnglearning.NewTest@91161c7]" name="cUnitTest" duration-ms="20" started-at="2016-08-11T12:13:01Z" finished-at="2016-08-11T12:13:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Manoj Kumar inside 'Unit Tests' tag]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cUnitTest -->
      </class> <!-- testnglearning.NewTest -->
    </test> <!-- Unit Tests -->
    <test name="Functional Tests" duration-ms="111" started-at="2016-08-11T06:43:01Z" finished-at="2016-08-11T06:43:01Z">
      <class name="testnglearning.SecondTest">
        <test-method status="PASS" signature="wFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.SecondTest@598067a5]" name="wFunctionalTest" duration-ms="21" started-at="2016-08-11T12:13:01Z" finished-at="2016-08-11T12:13:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Manoj Kumar inside 'Functional Tests' tag]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- wFunctionalTest -->
        <test-method status="PASS" signature="yFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.SecondTest@598067a5]" name="yFunctionalTest" duration-ms="20" started-at="2016-08-11T12:13:01Z" finished-at="2016-08-11T12:13:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Manoj Kumar inside 'Functional Tests' tag]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- yFunctionalTest -->
        <test-method status="PASS" signature="zFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.SecondTest@598067a5]" name="zFunctionalTest" duration-ms="20" started-at="2016-08-11T12:13:01Z" finished-at="2016-08-11T12:13:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Manoj Kumar inside 'Functional Tests' tag]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- zFunctionalTest -->
      </class> <!-- testnglearning.SecondTest -->
      <class name="testnglearning.NewTest">
        <test-method status="FAIL" signature="bFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.NewTest@3c0ecd4b]" name="bFunctionalTest" duration-ms="21" started-at="2016-08-11T12:13:01Z" finished-at="2016-08-11T12:13:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Manoj Kumar inside 'Functional Tests' tag]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at testnglearning.NewTest.bFunctionalTest(NewTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:178)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bFunctionalTest -->
        <test-method status="PASS" signature="dFunctionalTest(java.lang.String)[pri:0, instance:testnglearning.NewTest@3c0ecd4b]" name="dFunctionalTest" duration-ms="20" started-at="2016-08-11T12:13:01Z" finished-at="2016-08-11T12:13:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Manoj Kumar inside 'Functional Tests' tag]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dFunctionalTest -->
      </class> <!-- testnglearning.NewTest -->
    </test> <!-- Functional Tests -->
  </suite> <!-- Suite1 -->
</testng-results>
